apps/andi/assets/css/variables.scss

```diff
@@ -18,3 +18,6 @@ $table-padding: .5em 1.5em;
 $table-height: 2em;
 
 $box-shadow: 0 0 6px rgba(20, 20, 20, 0.2);
+
+$label-color: #7F7676;
+$border-color: #E2E5EA;
```

Comment:
What is the intended difference between `border-color` and `color-border`?  😁

Reply:
If you don't understand this intuitively, McSearchin, I'm not sure we can explain it to you.

(Seriously though good catch. I'm also curious).

Reply:
I'm just going to rip these variables out